clippy_lints/src/collapsible_if.rs

```diff
@@ -177,3 +201,17 @@ fn expr_block(block: &ast::Block) -> Option<&ast::Expr> {
         None
     }
 }
+
+/// If the expression is a `||`, suggest parentheses around it.
+fn par_around_or(expr: &ast::Expr) -> Vec<(Span, String)> {
```

Comment:
NIT: `parens` is a more common abbreviation of parentheses.

```suggestion
fn parens_around_or(expr: &ast::Expr) -> Vec<(Span, String)> {
```

Reply:
Done. I think I played with `Sugg::maybe_par()` first, and kept this abbreviation. I'll add checking if proposing to rename this to `Sugg::maybe_paren()` to my TODO list.